### ISSUE TITLE
Vive tracker tracks inconsistently at high speed

### ISSUE THREAD

**Issue by evolucid** [ForFunLabs/ElevenPublicBugsAndAsks#439]
I am using a Vive Tracker mounted on a Vive Paddle with Valve Base Stations.  They are Base Station 2's.  I have added two extra base stations for a total of four, but that did not alleviate the problem.  I was originally using Vive Tracker 2.0's (Same effect with three different ones), and I have now tried a Vive Tracker 3.0.  It seems like it is even worse with the Tracker 3.0.

The issue is only with the trackers and only with this game.  It is easily demonstrable by holding the paddle with my arm straight and twisting my wrist while keeping it in place.  It tracks accurately at slow speed, but if I speed up the twisting of my wrist the in-game paddle reacts wildly, flying out away from my wrist.

The result of this is that I can't swing the paddle fast at all.  If I do, the action of the in game paddle is randomized.  I have to just play defensively and redirect shots to be able to play effectively.  Now with the Tracker 3.0 (which is supposed to be more accurate), even that has a randomness to it.

**Comment by carrotstien** [comment]
hi evolucid. Unfortunately this is a problem with the lighthouse tracking
in general, and the trackers just exacerbate it. Any game that provides
more control for faster movement does so at the cost of a lot of assists or
success prediction or delayed behavior.
Typically the example people use is beat saber - and the answer is that it
the tracking skews there just as much, but the way it skews mostly just
makes your saber longer. That doesn't make you miss the cube...but for
table tennis, that skew is really bad. Stop by the discord and i'll be
happy to chat in more detail.
You can also read up a lot of work about this issue here
https://github.com/carrotstien/ElevenVRPlatformIssues/issues/1

discord link is first link of  : https://linktr.ee/elevenvr


TLDR: it's a steamvr problem. I've reached out to them multiple times and
they have given the proverbial shrug.

-Roman

**Comment by evolucid** [comment]
I did a quick test in SteamVR home, and the Tracker puck does exhibit the same unfortunate behavior there.  If there's anything I can do to help correct or mitigate this problem, please let me know.  This is my favorite game, and I don't think I could ever go back from using an actual paddle to play it.

**Comment by dukeoflegs** [comment]
Closing, no longer an issue

**Comment by dukeoflegs** [comment]
Reopened, awaiting information from SteamVR
